lib/ama/styles/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module AMA
   module Styles
-    VERSION = '3.2.1'
+    VERSION = '3.2.2'
```

Comment:
I believe for the version the last number is if it's for a bug, then minor change, then major change. So I think we actually want to be updating the middle number for these changes.